gravitee-apim-cypress/cypress/integration/bulk/subscriptions.test.ts

```diff
@@ -16,7 +16,7 @@
 
 import { gio } from '../../commands/gravitee.commands';
 import { fail } from 'assert';
-import { BasicAuthentication } from '../../model/users';
+import { BasicAuthentication } from '@model/users';
 import { Plan, PlanSecurityType } from '@model/plan';
 import { ADMIN_USER, API_PUBLISHER_USER } from 'fixtures/fakers/users/users';
```

Comment:
Might be better to have a uniform notation: Sometimes we use path alias (`@model/users`) and sometimes baseUrl (as in `fixtures/...`). 
I know this was not introduce by this change and might concern other files as well that's why it's just a reminder for a future PR.

Reply:
Yes, before this PR, we had mixed import notations.
Now, I think we should use imports with path aliases everywhere it's relevant.
I think I modified all files like that (also fixed the subscriptions.test.ts you mentioned here).
But I may have missed some. We have to keep in mind for subsequent PRs.